### PR TITLE
Fix broken `to_string` function

### PR DESCRIPTION
**Pull request by peaceful-james**

### CODE REVIEW

**Comment by peaceful-james** [comment]
@kipcole9 Nice library! This is a small fix with a small test but running `mix format` created a lot of changes. What version of Elixir are you using when developing this lib?

File: lib/cldr/person_name/formatter.ex [hunk 1]
```diff
@@ -722,12 +723,10 @@ defmodule Cldr.PersonName.Formatter do
...
     name_script = dominant_script(name)
-    name_locale = find_likely_locale_for_script(name_script, formatting_locale.backend)
 
-    if name_locale do
-      {:ok, name_locale}
-    else
-      {:error, "No locale resolved for script #{inspect(name_script)}"}
+    case find_likely_locale_for_script(name_script, formatting_locale.backend) do
+      {:ok, name_locale} when not is_nil(name_locale) -> {:ok, name_locale}
+      _ -> {:error, "No locale resolved for script #{inspect(name_script)}"}
```
Comment (peaceful-james) [inline lib/cldr/person_name/formatter.ex:729]: This is the actual fix. Everything else is mix format doing its thing.

**Comment by kipcole9** [comment]
Sorry for being slow on this, the PR is much appreciated.  I want to make sure I add a test case for this - do you happen to have the example that failed still around that I can use?

I am inconsistent in the use of the formatter - in the next round of CLDR updates I'll ensure formatting on push for all the libs.

**Comment by kipcole9** [comment]
I've pushed a commit that includes your change, thank you very much. I'll just wait to see if you have a previously failing test case before I publish.

And most of all - thanks for giving this library some love :-)

**Comment by peaceful-james** [comment]
@kipcole9 As well as I can remember, it was any name passed to `to_string`.

For example:

```
{:ok, name} = Cldr.PersonName.new(given_name: "James")
Cldr.PersonName.to_string(name)
```

That created a runtime exception because of a tuple somewhere that looked like `{:ok, {:ok, something}}`.

I will pull latest now and see if it works for me.

**Comment by peaceful-james** [comment]
Latest `main` branch has this issue already fixed. Closing this PR.

```
iex [1] > {:ok, name} = Cldr.PersonName.new(given_name: "James")
{:ok,
 %Cldr.PersonName{
   title: nil,
   given_name: "James",
   other_given_names: nil,
   informal_given_name: nil,
   surname_prefix: nil,
   surname: nil,
   other_surnames: nil,
   generation: nil,
   credentials: nil,
   preferred_order: nil,
   locale: nil
 }}
iex [2] > Cldr.PersonName.to_string(name)
{:ok, "James"}
```

**Comment by kipcole9** [comment]
I've added a bunch more tests (there were already tests but apparent;y they didn't catch this bug) and published an update to Hex.

**Pull request closed without merging**